RevenueCatUI/Resources/tr.lproj/Localizable.strings

```diff
@@ -1,19 +1,19 @@
 "OK" = "TAMAM";
 "All subscriptions" = "Tüm abonelikler";
-"Privacy" = "Mahremiyet";
+"Privacy" = "Gizlilik";
 "Privacy policy" = "Gizlilik Politikası";
 "Purchases restored successfully!" = "Satın almalar başarıyla geri yüklendi!";
-"Restore" = "Eski haline getirmek";
+"Restore" = "Geri yükle";
```

Comment:
Awesome thank you so much for improving these! 🙏🏻 

Just one question: is the longer one below still correct?
> "Restore purchases" = "Satın alınanları geri yükle";

Reply:
Thank you @NachoSoto 🙏🏻 It seems accurate and meaningful. If a better and clearer wording comes to my mind, I'll let you know.

Reply:
Awesome thanks again for your contribution!

Reply:
Hi @iremkaraoglu, what do you think `"Satın almaları geri yükle"`? Because it is used as "satın alma" in this line, I thought it would be better if it was the same on both sides.
`"Purchases restored successfully!" = "Satın almalar başarıyla geri yüklendi!";`

Reply:
Hey @mavideniz, I see your point. Yes, it makes sense. 🙌🏻